### ISSUE TITLE
Updating TSP100IV Firmware to support CloudPRNT Version MQTT

### ISSUE THREAD

**Issue by JTimings** [star-micronics/cloudprnt-sdk#11]
It seems that the TSP100IV possesses all the necessary hardware to support CloudPRNT MQTT. Is there a plan to introduce CloudPRNT MQTT to the TSP100IV and other existing CloudPRNT printers? The HTTP polling method is terribly inefficient, but the cost of upgrading existing printers to the mC-Print3 just for MQTT support is prohibitive.

**Comment by bandit-ibayashi** [comment]
@JTimings Thank you for your opinion. We understood your point.
There is nothing clear to say about MQTT support in the TSP100IV-UE, but will let you know if there are any updates on this matter.

Thank you for your understanding.

**Comment by holtkamp** [comment]
We have a TSP100IV (`CloudPRNT/5.1 TSP100IV/2.2`) running on firmware `TPS100IV_v22(MAIN)_V22(BOOT).mot` that issues a HTTP GET request to  `<CloudPRNTServerURI>/cloudprnt-setting.json` every hour. 

When searching for `cloudprnt-setting.json`, I encountered: https://github.com/star-micronics/starlabs-cloudprnt-sample-server-php-aws?tab=readme-ov-file#setting-cloudprnt-settingjson

> Setting cloudprnt-setting.json
> Edit the json file below according to the CloudPRNT communication protocol you want to use.
> This json file is used as a server response to [Server setting information request (GET)](https://star-m.jp/products/s_print/sdk/StarCloudPRNT/manual/en/protocol-reference/http-method-reference/server-info-get/index.html).

https://star-m.jp/products/s_print/sdk/StarCloudPRNT/manual/en/protocol-reference/http-method-reference/server-info-get/index.html

Which states: 

> Only printers that support CloudPRNT Version MQTT execute this HTTP method.

Does this indicate that the https://star-emea.com/products/tsp100iv/ supports MQTT? 
If not, can `cloudprnt-setting.json` still be used to update printer settings at run time?

**Comment by bandit-ibayashi** [comment]
@holtkamp Yes, TSP100IV has started supporting MQTT with CloudPRNT from firmware version 2.2. As a result, the process of retrieving `cloudprnt-setting.json` file via GET is now being performed. If the file cannot be retrieved, the conventional HTTP polling process will be used instead.

@JTimings We sincerely apologize for not notifying you at the time of MQTT compatibility. By updating the TSP100IV to the latest firmware, you will be able to utilize MQTT. The firmware update can be carried out using our application, Star Quick Setup Utility, or Star Windows Software.
